lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,124 @@
+var edx = edx || {},
+
+    Dropdown = (function() {
+
+        var _dropdown = {
+
+            opts: {
+                page: $(document),
+                midpoint: ($(window).width() / 2),
+                button: $('.button-more.has-dropdown'),
+                button_active: 'is-active',
+                menu: $('.dropdown'),
+                menu_inactive: 'is-hidden',
+                menu_active: 'is-visible',
+                menu_align: 'align-',
+                text_align: 'control-text-align'
+            },
+
+            init: function() {
```

Comment:
Can this be updated to accept a parent element, so that finds can be scoped to be within said parent? This will make it more useful when using Backbone / RequireJS as well as being slightly faster (because it won't have to look through the whole of the DOM)